packages/js/src/ui/components/elements/Header/PreferencesHeader.tsx

```diff
@@ -12,15 +13,19 @@ export const PreferencesHeader = (props: PreferencesHeaderProps) => {
 
   return (
     <div class={style('preferencesHeader', 'nt-flex nt-items-center nt-py-5 nt-px-6 nt-gap-2')}>
-      <button
-        class={style(
-          'preferencesHeader__back__button',
-          'nt-h-6 nt-w-6 nt-flex nt-justify-center nt-items-center nt-rounded-md nt-relative hover:nt-bg-foreground-alpha-50 focus:nt-bg-foreground-alpha-50 nt-text-foreground-alpha-600'
+      <Show when={props.navigateToNotifications}>
+        {(navigateToNotifications) => (
+          <button
+            class={style(
+              'preferencesHeader__back__button',
+              'nt-h-6 nt-w-6 nt-flex nt-justify-center nt-items-center nt-rounded-md nt-relative hover:nt-bg-foreground-alpha-50 focus:nt-bg-foreground-alpha-50 nt-text-foreground-alpha-600'
+            )}
+            onClick={navigateToNotifications()}
```

Comment:
```suggestion
            onClick={navigateToNotifications}
```
❓ **question:** should it be like that?

Reply:
If we use `createMemo`, I believe so.